wyvern/exceptions.py

```diff
@@ -95,3 +95,7 @@ class ExperimentationProviderNotSupportedError(WyvernError):
 
 class ExperimentationClientInitializationError(WyvernError):
     message = "Failed to initialize experimentation client for provider: {provider_name}, {error}"
+
+
+class EntityColumnMissingError(WyvernError):
```

Comment:
There's also this exception. 

```
class WyvernEntityValidationError(WyvernError):
    message = "{entity_key} is missing in entity data: {entity}"
```

Reply:
They're a bit different though. WyvernEntityValidationError is used in the indexation endpoints.